packages/api-explorer/schema-form.css

```diff
@@ -0,0 +1,297 @@
+form.rjsf .field label:after {
```

Comment:
Since these style rules are pretty heavy handed (i.e. lots of `!important`...) I've kept all my CSS scoped everything within the `form.rjsf` selector.

Reply:
We already have this file: https://github.com/readmeio/api-explorer/blob/master/packages/api-explorer/api-explorer.css. Can we merge the two?

Reply:
If you want to take this as an opportunity to move us over to Sass... 😬

Reply:
Word up. Let’s do it.

Reply:
Done deal my dudes! SCSS it is...

Reply:
![giphy-downsized](https://user-images.githubusercontent.com/33762/65922080-fb2bda00-e398-11e9-95c3-d78851be395a.gif)

Reply:
@erunion lolololol 🕶 back atchya my man